Extraction Error Log
Timestamp: 2025-08-11T14:09:17.396330
Error: name 'total_records' is not defined
Traceback:
Traceback (most recent call last):
  File "/var/folders/mr/n19psg616pb4dyc16wbyn8yc0000gn/T/ipykernel_2464/3884050967.py", line 81, in <module>
    'total_records': total_records,
                     ^^^^^^^^^^^^^
NameError: name 'total_records' is not defined

